src/viewer_window.cpp

```diff
@@ -372,6 +386,13 @@ class DatarefViewerWindow {
 		last_regex = 0 != XPGetWidgetProperty(regex_toggle_button, xpProperty_ButtonState, nullptr);
 		last_changed = 0 != XPGetWidgetProperty(change_filter_button, xpProperty_ButtonState, nullptr);
 
+        FILE * pFile;
+        pFile = fopen ("./Resources/plugins/DataRefTool/drtpref.txt","w+");
```

Comment:
Shouldn't we use Xplane's XPLMGetPrefsPath() to figure out the preferred directory?